### DISCUSSION TITLE
How to use SD3 Medium model in Foocus

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#3142] [Q&A] [answered]
https://stability.ai/news/stable-diffusion-3-medium

I 've cheched there came out new model.

Please let me know how to test this model in Fooocus.

Thanks.

**Comment by chouhai2018** [comment]
我也特别想知道,期待中

**Comment by ILikeToasters** [comment, marked as answer]
Fooocus does not support it yet. Honestly it is going to take some time before SD3 can come close to what SDXL can do right now from what I have seen from my testing of it so far.

**Reply by oldhand7** [reply]
I don't know the main difference between SD3 and sdxl. Can we use it by putting model in checkpoint folder?

**Reply by ILikeToasters** [reply]
No. There will need to be changes made to support it. Honestly at the moment you would probably not be happy with the results you would get with it especially with generating images of people. I have used it on several other UIs and it really is not that great right now.

**Reply by chouhai2018** [reply]
底层模型照片人像确实不怎么样,漫画还行,空间物体等挺好,SD3的细节很好,这个底层模型上面训练,会非常好的.

**Reply by PeterJared** [reply]
From detail and lack of detail artifacts SD3 seems superior to me to most of any other image generation. This testing with the Stable Assistant, that seem higher quality that the API initial demo they enabled. 
Also the expand canvas feature seems to me the best in the market in terms of being coherent.  They rest AI applications make this rather messy when this is nearer to the original image quality.
Text is better than older versions but worse than Ideogram.
Visual quality is very good even a bit difficult to control from style perspective, in that sense not to the level of older versions. But I'm nearly sure when the developers add new checkpoints and is properly supported by Fooocus and others, could be the best AI generator.

**Reply by oldhand7** [reply]
Hi,everyone, thank you for your advice. I have a question. 
![image](https://github.com/lllyasviel/Fooocus/assets/91205658/d22aebca-e5a1-46d5-aaa5-fb0f2a2c593d)


I 've seen from discord channel.

I am just curious how to check this performance and quality of this  model - where, which engine. 

Thank you for your help.

**Comment by xhoxye** [comment]
You need to wait for the code update for fooocus

**Reply by IPv6** [reply]
There is also a problem of supplementary models (controlnets, inpainting). Someone have to train them, train well

**Comment by graz68a** [comment]
Anyone is able to have good results with the prompt below used by Stable Diffusion medium 3 (using Fooocus) ?

_photo of three antique dragon glass magic potions in an old abandoned apothecary shop: the first one is blue with the label "1.5", the second one is red with the label "SDXL", the third one is green with the label "SD3"._
![Stable_Diffusion_3_Medium](https://github.com/lllyasviel/Fooocus/assets/28712630/8a35b2f8-c794-4a1a-a17e-19899276cd5b)

I tried various models in Fooocus but no one seems to be able to have this perfect result.

**Reply by AnalyticsXXI** [reply]
It looks like SD 3 just can't deal with people, but quite impressive in the other areas.

**Reply by PeterJared** [reply]
why do you think so? 
![image](https://github.com/lllyasviel/Fooocus/assets/49351356/8f0f50d0-1794-4877-8468-d09893181c01)
![image](https://github.com/lllyasviel/Fooocus/assets/49351356/0d822c73-60f5-419a-bf4b-76bb65efacaf)
![image](https://github.com/lllyasviel/Fooocus/assets/49351356/b10f86f5-cc17-4d5b-b248-cc02fabc513b)

**Reply by southbayjay** [reply]
> Anyone is able to have good results with the prompt below used by Stable Diffusion medium 3 (using Fooocus) ?
> 
> _photo of three antique dragon glass magic potions in an old abandoned apothecary shop: the first one is blue with the label "1.5", the second one is red with the label "SDXL", the third one is green with the label "SD3"._ ![Stable_Diffusion_3_Medium](https://private-user-images.githubusercontent.com/28712630/339324473-8a35b2f8-c794-4a1a-a17e-19899276cd5b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgzMTgyNTIsIm5iZiI6MTcxODMxNzk1MiwicGF0aCI6Ii8yODcxMjYzMC8zMzkzMjQ0NzMtOGEzNWIyZjgtYzc5NC00YTFhLWExN2UtMTk4OTkyNzZjZDViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEzVDIyMzIzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVjYTgwNmJmYzkwZDllNDNlM2FjM2FiZTIyZTRmM2FkYjg4N2Y5ZGMxOTgyYTFjZjUwZDY2M2YwZGRkODE0NmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6gWEhMaRbLK1K8DRJ9zeGUEYtDbwqfrhN59FVuNPTPM)
> 
> I tried various models in Fooocus but no one seems to be able to have this perfect result.

Did you do this in Fooocus with SD3??? If so, how???

**Reply by AnalyticsXXI** [reply]
> why do you think so?

I don't mean that it never can generate any decent human face. With some luck, it can even generate non-nightmare human anatomy, but overall working with humans in SD3 doesn't look promising at all, in compare with other tools we have today. And its license will probably seriously slow down any progress of fixing it.

Initially I was hyped out about SD3, but now I don't even see a reason to hurry with implementing it in Fooocus, almost don't care anymore. Landscapes are great though.

**Reply by chivoyage** [reply]
> why do you think so? ![image](https://private-user-images.githubusercontent.com/49351356/339488118-8f0f50d0-1794-4877-8468-d09893181c01.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkzMjY0MjMsIm5iZiI6MTcxOTMyNjEyMywicGF0aCI6Ii80OTM1MTM1Ni8zMzk0ODgxMTgtOGYwZjUwZDAtMTc5NC00ODc3LTg0NjgtZDA5ODkzMTgxYzAxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI1VDE0MzUyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ4OTI4ZDZiYTBkNzViOThkMDA3ODE5Yjg4MTk0MTY4MjI3OTFkOGM1YzVlMTZmMGZhODE5M2U1MzMwOTU0NzcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bvH4tOA3HH6v-Vz-J0LL2yWAwq5mX9jmQ5mK5AxE4l0) ![image](https://private-user-images.githubusercontent.com/49351356/339489146-0d822c73-60f5-419a-bf4b-76bb65efacaf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkzMjY0MjMsIm5iZiI6MTcxOTMyNjEyMywicGF0aCI6Ii80OTM1MTM1Ni8zMzk0ODkxNDYtMGQ4MjJjNzMtNjBmNS00MTlhLWJmNGItNzZiYjY1ZWZhY2FmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI1VDE0MzUyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMxY2NmMzQyZWQ3NDFkZjJkNzQ5MDAxNDMwOTgyZTk2MzI5OGQyNzc5NDRjN2ZjNGMxYzRhODFjY2I3OGJmM2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._tY7cH3IjD2fcdyVOgxY_1No68DAkwal1pqGA2UQVhU) ![image](https://private-user-images.githubusercontent.com/49351356/339489842-b10f86f5-cc17-4d5b-b248-cc02fabc513b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTkzMjY0MjMsIm5iZiI6MTcxOTMyNjEyMywicGF0aCI6Ii80OTM1MTM1Ni8zMzk0ODk4NDItYjEwZjg2ZjUtY2MxNy00ZDViLWIyNDgtY2MwMmZhYmM1MTNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI1VDE0MzUyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRhMzlhNjE1YTMzY2Q2NjUxYjFlOThlNTNlNTFlM2U3MDIyMTAzMjBmMjc3NDc3NThkMWIyODkzMDI5MWVlNDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.q7mpvNfhEIK_WvB3pcEP6GTFsOaPxn6mxkZDjuNHroc)

Is this on Fooocus with SD3?
